### ISSUE TITLE
Installation right

### ISSUE THREAD

**Issue by AleksDominik** [leriel/pyspark-easy-start#2]
seems that pyspark is not installed anymore on the image
I also tried to make a pipenv env  but permission denied on ./local
sudo chmod answer is sudo not found
any ideas?

**Comment by Don-Burns** [comment]
If you run the following it should allow you to create a new image with pyspark which you can point the leader in the docker-compose to.
`docker run -ti -u root -v $(pwd)/test.py:/bitnami/spark/test.py bitnami/spark:3-debian-10 bash`
in the interactive shell
`pip install pyspark`
Once this is complete `exit` and `docker container ls` find the container ID which you were just in.
execute docker `commit {container-id} pyspark-bitnami`
then edit the docker file on line 30 to be `image: pyspark-bitnami`
You should then be able to follow steps as normal in the README

**Comment by leriel** [comment]
@AleksDominik @Don-Burns Current `docker.io/bitnami/spark:3-debian-10` image resolves to spark 3.2.0, which has different py4j path, but also has some other issues. Please lock image down to docker.io/bitnami/spark:3.1.2, eg
```
services:
  spark:
    image: docker.io/bitnami/spark:3.1.2
    environment:
      - SPARK_MODE=master
      - SPARK_RPC_AUTHENTICATION_ENABLED=no
      - SPARK_RPC_ENCRYPTION_ENABLED=no
      - SPARK_LOCAL_STORAGE_ENCRYPTION_ENABLED=no
      - SPARK_SSL_ENABLED=no
    volumes:
      - .:/app
    ports:
      - '127.0.0.1:8080:8080'
```
This worked for me, let me know if it works for you as well.

I'll update the repo with fixed images

**Comment by AleksDominik** [comment]
I rebuild a new image from scratch.
The way this image was build block su utilisation unless specifying on dockerfile.
The other problem was the python version, that didnt fit with my need.
you can find a small fork https://github.com/AleksDominik/bitnami-docker-spark. with python 3.7.
I also build a docker image lesakmidonik/pysparkcluster:py37 hope it will help. 
I dont really documented anything actually. I will do it asap.
